src/style.scss

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+$--blue-color: #00acdc;
+$--gray-color: #616070;
+$--black-color: #060b35;
+$--white-color: #fff;
+$--standard-width: 166px;
+$--bold-font-weight: 700;
+$--medium-font-weight: 500;
+$--regular-font-weight: 400;
+
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
```

Comment:
```suggestion

```
Avoid limiting the height of the card, it should depend on the content in this case